npm/vite-plugin-cypress-esm/cypress/component/ignoreModuleList.cy.ts

```diff
@@ -1,7 +1,8 @@
-import * as Immutable from './fixtures/MyImmutableModule'
```

Comment:
Could we leverage existing utility methods instead of introducing new ones here?